docs/html/development/release-process.rst

```diff
@@ -57,6 +57,21 @@ their merits.
   ``pip._internal.utils.deprecation.deprecated``. The function is not a part of
   pip's public API.
 
+Python 2 support
+----------------
+
+pip will continue to ensure that it runs on Python 2.7 after the CPython 2.7
+EOL date. Support for Python 2.7 will be dropped, if bugs in Python 2.7 itself
+prevent this (which is unlikely) or Python 2 usage reduces to a level that pip
```

Comment:
prevent this -> make this necessary
reduces to a level that pip... ->  reduces to a level where the pip...